package.json

```diff
@@ -67,6 +67,7 @@
     "docs:fix": "remark --output -- './{,!(node_modules)/**/}!(CHANGELOG)*.md'",
     "docs:lang": "node build/translations.js",
     "netlify": "babel-node ./build/netlify-docs.js",
+    "prepare": "patch-package",
```

Comment:
prepare runs before packing the package when when `npm install` is run without anything https://docs.npmjs.com/misc/scripts